mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -408,6 +408,54 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 								},
 							},
 						},
+
+						"stack_type": {
```

Comment:
Can we test and document this for instance template or switch to guarding the shared code (eg by passing in a mode var) instead? I'd be concerned about adding fields that aren't actually supported- it takes one person going through the fields and not clicking `git blame` to find the context, and all of the sudden they're in our docs.

Reply:
Sure! Thanks! One thing I noticed is, the InstanceTemplate REST API docs show that these are all supported, however, I didn't see the computed fields returned (or even when I tried to set `ipv6_access_config.public_ptr_domain_name`). It accepted it to be set, but didn't return it, so I checked what `access_config` had and it's just `Computed` there, with a note, so I just copied that over and did the same thing. I also didn't mention the outputs in the docs, simply because I didn't see them returned - despite the documentation saying they should be? I can remove them if you'd prefer, or just leave them for compatibility with Instance. They shouldn't hurt anything, right? We might just have to add documentation later.

Reply:
> I also didn't mention the outputs in the docs, simply because I didn't see them returned - despite the documentation saying they should be

Ah- since it's not actually provisioned yet, I think this probably makes sense. Thanks! Seems like the right call to me.